### PR TITLE
feat(route): 新增中国有色金属工业网路由

### LINKED ISSUE #16748: 【景气指数 & 行业分析 & 数据统计  中国有色金属工业协会】
### 类型

金融

### 网站地址

https://www.chinania.org.cn/

### 网站描述

中国有色金属工业协会（China Non-Ferrous Metals Industry Association）是由我国有色金属行业的企业、事业单位、社会团体为实现共同意愿而自愿组成的全国性、非营利性、行业性的经济类社会团体，是依法成立的社团法人。

业务范围：
中国有色金属工业协会的业务范围共十项：
一、根据国家政策法规制定并监督执行行规行约，规范行业行为，协调同行价格争议，维护公平竞争。

二、通过调查研究为政府制定行业发展规划、产业政策和有关法律法规提出意见和建议。

三、协助政府主管部门制定、修订本行业国家标准，负责本行业标准的制定、修订和实施监督。

四、根据政府有关部门的授权和委托，开展行业统计调查工作，采集、整理、加工、分析并发布行业信息。

五、根据政府有关部门的授权和委托，对基建、技术改造、技术引进、投资与开发项目进行前期论证。

六、根据政府有关部门的授权和委托，参与质量管理监督工作和资质审查，开展行检、行评，承担生产、经营许可证审查，开展行业损害调查工作。

七、组织科技成果鉴定、评奖与推广应用，开展职称评定、人才交流、业务培训，创办刊物，咨询服务，组织展销会、展览会。

八、组织行业的国际经济技术交流与合作，负责国际双边或多边各类协议的组织和实施，参加国际同业组织的活动。

九、反映会员要求，协调会员关系，维护会员合法权益。

十、承担政府有关部门委托的其他工作。

https://www.chinania.org.cn/html/xiehuigaikuang/xiehuijianjie/

### 需要生成什么内容？

景气指数  https://www.chinania.org.cn/html/hangyetongji/jqzs/

行业分析  https://www.chinania.org.cn/html/hangyetongji/tongji/

数据统计 https://www.chinania.org.cn/html/hangyetongji/chanyeshuju/

### 额外描述

感谢

### 这不是重复的 RSS 请求

- [X] 我已经搜索了[现有 issue](https://github.com/DIYgod/RSSHub/issues) 和 [pull requests](https://github.com/DIYgod/RSSHub/pulls)，以确保该 RSS 尚未被请求。

### PR DESCRIPTION
**Pull request by murphysking**
## Involved Issue / 该 PR 相关 Issue
Close #16748 

## Example for the Proposed Route(s) / 路由地址示例

```routes
/chinania/xiehuidongtai/xiehuidongtai
/chinania/xiehuidongtai/xiehuitongzhi
/chinania/xiehuidongtai/youseqiye50qiang
/chinania/djgz/xiehuidangjian
/chinania/djgz/hangyedangjian
/chinania/hangyexinwen/shizhengyaowen
/chinania/hangyexinwen/yaowen
/chinania/hangyexinwen/guoneixinwen
/chinania/hangyexinwen/zixun
/chinania/renliziyuan/xiangguantongzhi
/chinania/renliziyuan/renshizhaopin
/chinania/hangyetongji/tongji
/chinania/hangyetongji/chanyeshuju
/chinania/hangyetongji/jqzs
/chinania/zcfg/zhengcefagui
/chinania/hyzl/huiyizhanlan
/chinania/hyzl/huizhanbaodao
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [x] New Route / 新的路由
  - [x] Follows [Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [x] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [x] Parsed / 可以解析
  - [ ] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

### CODE REVIEW

**Comment by KwToPA** [comment]
应该是网站封杀境外IP

File: lib/routes/chinania/index.ts [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+                    url: item.link,
+                });
+                const content = load(detailResponse.data);
+                item.description = item.link.includes('www.news.cn') ? content('#detailContent').html() : content('.article_content').html();
+                return item;
+            })
+        )
+    );
+    return {
+        title: '中国有色金属工业网-' + typeenum[type] + '-' + subtypeenum[subtype],
```
Comment (TonyRL) [inline lib/routes/chinania/index.ts:79]: Both values can be obtained programmatically, which does not require hard-coding.
Major type name can be obtained using `$('.nav_title').text()` while minor type can be obtained using `$('.notice_title_left span').text()`

File: lib/routes/chinania/index.ts [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+        link: currentUrl,
+        description,
```
Comment (TonyRL) [inline lib/routes/chinania/index.ts:81]: Do not fill the [description of the feed](https://www.rssboard.org/rss-specification#requiredChannelElements) with the Markdown table https://github.com/DIYgod/RSSHub/blob/183642a654f903cfec6f7223b7cd2cf77643c4ca/lib/routes/chinania/index.ts#L8

File: lib/routes/chinania/index.ts [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+    path: '/:type/:subtype',
+    categories: ['government'],
+    description,
+    radar: [
+        {
+            source: ['www.chinania.org.cn/html/:type/:subtype'],
+            target: '/:type/:subtype',
+        },
+    ],
+    name: '中国有色金属工业网',
```
Comment (TonyRL) [inline lib/routes/chinania/index.ts:97]: Route name shouldn't be the same as namespace name.

File: lib/routes/chinania/index.ts [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+    maintainers: ['murphysking'],
+    handler,
+    url: 'www.chinania.org.cn/',
+    example: '行业统计-行业分析  `hangyetongji/tongji`',
```
Comment (TonyRL) [inline lib/routes/chinania/index.ts:101]: example must start with `/` with the namespace prepended

**Pull request closed without merging**